### PR TITLE
Make registry yaml export more suitable for diffing/merging

### PR DESCRIPTION
**Pull request by rvl**

### CODE REVIEW

File: docker-compose-build.yml [hunk 1]
```diff
@@ -43,8 +39,8 @@ services:
       context: ./
       dockerfile: Dockerfile-prod
       args:
-        ARG_GIT_TAG: ${GIT_TAG}
-    image: muccg/rdrf:${GIT_TAG}
+        ARG_GIT_TAG: asdf
+    image: muccg/rdrf:consent-filename
```
Comment (id2359) [inline docker-compose-build.yml]: This looks like a change was accidentally commited?

Comment (rvl) [inline docker-compose-build.yml]: fixed

**Comment by id2359** [comment]
approved

**Comment by id2359** [comment]
approved

**Pull request merged**